app/components/Navbar.jsx

```diff
@@ -13,24 +13,24 @@ class Navbar extends React.Component {
 
   render() {
     return (
-      <nav style= {{backgroundColor: '#ADD8E6'}} >
-      <div>This is the NavBar Fix Me Later</div>
-      <ul className="nav navbar-nav navbar-right">
-        <li>
-          <Link to="/products" >Bicycles</Link>
-        </li>
-        <li>
-          <Link to="/authenticate" >Sign Up</Link>
-          {this.props.user ? <WhoAmI/> : <Login/>}
-        </li>
-        <li>
-          <Link to="/cart">Cart</Link>
-        </li>
-        <li>
-          <Link to="/users">Admin Only: All Users</Link>
-        </li>
-      </ul>
-
+      <nav className="navbar navbar-inverse">
+        <ul className="nav navbar-nav">
+          <li>
```

Comment:
Gabe, good job! Just a minor opinion: 
what do you think about a small "Lisa's Bike" button on the navbar which leads to the home page so that the user can easily navigate back to the home page? '-'